tests/jvm/src/test/resources/scalalib.expect

```diff
@@ -126,3 +126,19 @@ scala.Null# => abstract final class Null.{+1 decls}
   extends AnyRef
 scala.Null#`<init>`(). => primary ctor <init>: (): Null
   Null => scala.Null#
+
+scala/Singleton.class
+---------------------
+
+Summary:
+Schema => SemanticDB v3
+Uri => scala/Singleton.class
+Text => empty
+Language => Scala
+Symbols => 2 entries
+
+Symbols:
+scala.Singleton# => final trait Singleton
+  extends Any
+scala.Singleton#`<init>`(). => primary ctor <init>: (): Singleton
```

Comment:
Why is the constructor here? I thought the guard against `k.CLASS` prevented traits from having constructors.

Reply:
Not sure about this either: the interesting thing is that the `Singleton` trait doesn't have own declarations anymore (`{+1 decls}` is gone). Could that constructor be inherited from the `Any` class?

Reply:
I suggest we remove the constructor here. The spec says "Unlike some other metaprogramming systems for Scala, we do not create synthetic constructor symbols for traits and objects".

Reply:
Sure, I had tried already, but I've missed an `if` on the top-level info symbols.